### DISCUSSION TITLE
Does biome support custom HTML-based template syntax?

### DISCUSSION THREAD

**Discussion by Th3S4mur41** [biomejs/biome#372] [Q&A] [answered]
Certain popular frameworks come with their own HTML-based template syntax and their own extension...

Does biome support extensions like [.vue](https://vuejs.org/guide/essentials/template-syntax.html), [.astro](https://docs.astro.build/en/core-concepts/astro-components/) and [.svelte](https://svelte.dev/docs/basic-markup)

**Comment by Conaclos** [comment, marked as answer]
Not yet. We have first to support CSS (work in progress) and HTML.
